frontend/packages/ceph-storage-plugin/src/components/block-pool/block-pool-menu-action.ts

```diff
@@ -0,0 +1,24 @@
+import i18next from 'i18next';
+
+import { asAccessReview } from '@console/internal/components/utils';
+import { K8sKind, K8sResourceKind } from '@console/internal/module/k8s';
+
+import { updateBlockPoolModal } from '../modals/block-pool-modal/update-block-pool-modal';
+import { deleteBlockPoolModal } from '../modals/block-pool-modal/delete-block-pool-modal';
+
+const editBlockPool = (kind: K8sKind, blockPoolConfig: K8sResourceKind) => ({
+  labelKey: i18next.t('ceph-storage-plugin~Edit BlockPool'),
```

Comment:
in console labels are externalize this way. i18next works only on refresh.
```suggestion
  // t('ceph-storage-plugin~Edit BlockPool')
  labelKey: t('ceph-storage-plugin~Edit BlockPool'),
```

Reply:
Or of there is some way you an pass "t" function to this

Reply:
ack, try to pass this in customData